### DISCUSSION TITLE
Five(server) 404 error page not found error

### DISCUSSION THREAD

**Discussion by SomyaAgr** [yandeu/five-server#108] [Help / Questions]
@yandeu I m coding in a virtual space and using remote repository extension while I'm try to go live with five server, I'm getting 404 error 
here i m screenshot of my vsc 
![image](https://github.com/yandeu/five-server/assets/91428207/301203b8-971a-4011-887f-ce8296155d61)
and also the screenshot of error i m getting 404 error
![image](https://github.com/yandeu/five-server/assets/91428207/8b3ee8d6-8587-456e-9d7d-3c1b2afb04b8)

Thanks in advance

**Comment by yandeu** [comment]
Can you tell me more about your environment (virtual space and using remote repository)?

**Comment by SomyaAgr** [comment]
Yea sure
So it's like I am working on my repository without cloning it I am using it
virtually
Their I m trying to use five server and couldn't do anything for now

**Reply by yandeu** [reply]
Unfortunately, I have no idea what this means.

**Comment by jplanza** [comment]
@yandeu I'm seeing the same issue with VS Code for Linux; the server starts and opens a browser window, but every page returns a 404.  Here's what I know so far:

- File permissions look good, the directory is 775 and the files are 664; all are owned by my user
- I started "Go Live" from a folder
- The extension clearly works; as I click on files in the VS Code UI, the browser window is updating
- The content is available locally; I have the repo checked out on a branch.
- VS Code file type detection marks these files as "Django HTML" (as expected)

 I don't understand the comments from the OP either, but I see the same symptoms and should be able to gather more information for you.  Please let me know what you need.
